symplyphysics/laws/condensed_matter/resistance_from_temperature.py

```diff
@@ -0,0 +1,46 @@
+from sympy import (Eq, solve)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The resistance depends on the temperature. For different materials, the value
+## of the temperature coefficient and resistance at zero degrees celsius may differ.
+
+## Law is: R = R0 * (1 + a * T), where
+## R - resistance,
+## R0 - resistance at zero degrees celsius,
+## a - temperature coefficient,
+## T - temperature change.
+
+resistance = Symbol("resistance", units.impedance)
+
+resistance_initial = Symbol("resistance_initial", units.impedance)
+temperature_coefficient = Symbol("temperature_coefficient", 1 / units.temperature)
+temperature = Symbol("temperature", units.temperature)
```

Comment:
It's temperature change, or temperature difference from zero degrees celsius. To avoid long name for this variable, maybe it is better to use difference of two temperatures?